libs/design-system/cards/count-card/src/lib/count-card.component.stories.ts

```diff
@@ -0,0 +1,36 @@
+import { Meta, StoryObj } from '@storybook/angular';
+import { CountCardComponent } from './count-card.component';
+
+const meta: Meta<CountCardComponent> = {
+  component: CountCardComponent,
+  title: 'Design System/Cards/Count Card',
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/gQEMLugLjweDvbsNNUVffD/HRA-Design-System-Repository?node-id=8607-27441',
+    },
+  },
+  args: {
+    count: 250,
+    showSuffix: true,
+    label: 'label text',
+    iconType: 'category',
+    icon: 'data',
+  },
```

Comment:
Add argTypes for select control:

`
argTypes: {
    iconType: {
      control: 'select',
      options: ['category', 'product', 'organ'],
    },
 }
`